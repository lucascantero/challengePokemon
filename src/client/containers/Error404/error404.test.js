import React from 'react';
import Error404 from './Error404.jsx';
import { act } from 'react-dom/test-utils';
import {render,unmountComponentAtNode} from 'react-dom';


describe('Testing Error 404', () => {

    let container = null;

    beforeEach(() => {
        container = document.createElement("div");
        document.body.appendChild(container);
    });

    afterEach(() => {
        unmountComponentAtNode(container);
        container.remove();
        container = null;
    });


    it('Should match snapshot', () => {
        
        act(() => {
            render(<Error404 />, container)
        })

        expect(container.innerHTML).toMatchSnapshot();
    })
})