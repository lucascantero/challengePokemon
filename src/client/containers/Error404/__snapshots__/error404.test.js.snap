// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Error 404 Should match snapshot 1`] = `"<h1>Error 404</h1>"`;
